shell/common/shell.h

```diff
@@ -78,6 +78,8 @@ class Shell final : public PlatformView::Delegate,
 
   DartVM* GetDartVM();
 
+  void PurgeCaches() const;
```

Comment:
Please add docstring.  You might want to note that it is an asynchronous call.

Reply:
lgtm, one nagging thought I have in the back of my mind is that iOS might not like the fact that this is asynchronous, the process might get killed if no positive action doesn't happen as a result of the notification.  We can just keep an eye on it though.

Reply:
Agreed but this should hopefully improve things.  I'm going to separately look to reduce the cache size based on available memory.